index.js

```diff
@@ -70,22 +71,17 @@ app.post('/github-hook', function (req, res, next) {
             } catch(e) {
                 return;
             }
-            if (!body) {
-                logArgs("Ignoring event: empty body");
-                return;
-            }
-            if (body.sender && body.sender.login == "wpt-pr-bot") {
-                logArgs("Ignoring event: sender is wpt-pr-bot");
+            if (!filter.event(body, logArgs)) {
                 return;
             }
             if (!body.pull_request && (body.issue && !body.issue.pull_request)) {
                 logArgs("Ignoring event: not a pull request");
                 return;
             }
-            if (body.pull_request && body.pull_request.draft) {
-                logArgs("Ignoring event: pull request is a draft");
+            if (!filter.pullRequest(pull_request, logArgs)) {
```

Comment:
`pull_request` is undefined.

JSHint is capable of detecting these errors. This pull request's "passing"
status demonstrates that we're not currently linting `index.js`. I've opened
gh-87 to address that.

Reply:
Rebased and fixed.